web/src/compose_validate.js

```diff
@@ -230,38 +230,39 @@ export function warn_if_topic_resolved(topic_changed) {
     }
 }
 
-function show_wildcard_warnings(stream_id) {
+function show_wildcard_warnings(stream_id, $banner_container) {
     const subscriber_count = peer_data.get_subscriber_count(stream_id) || 0;
+    const stream_name = compose_state.stream_name() || stream_data.maybe_get_stream_name(stream_id);
 
     const classname = compose_banner.CLASSNAMES.wildcard_warning;
     const wildcard_template = render_wildcard_warning({
         banner_type: compose_banner.WARNING,
         subscriber_count,
-        stream_name: compose_state.stream_name(),
+        stream_name,
         wildcard_mention,
         button_text: $t({defaultMessage: "Yes, send"}),
         hide_close_button: true,
         classname,
     });
 
     // only show one error for any number of @all or @everyone mentions
-    if ($(`#compose_banners .${CSS.escape(classname)}`).length === 0) {
-        compose_banner.append_compose_banner_to_banner_list(
-            wildcard_template,
-            $("#compose_banners"),
-        );
+    if ($banner_container.find(`.${CSS.escape(classname)}`).length === 0) {
+        compose_banner.append_compose_banner_to_banner_list(wildcard_template, $banner_container);
```

Comment:
We likely need to do something so the button says "Yes, save" when editing.

Reply:
I think it would be appropriate, since clicking on the button saves the message, even though in the end we send it and use the "send" word in the main message.
![image](https://user-images.githubusercontent.com/53193850/236433834-c27efbd9-e21e-4085-87a2-73242d7f81d1.png)

Reply:
Sounds good.